configs/config-suite-b.h

```diff
@@ -32,6 +32,7 @@
 #define MBEDTLS_SSL_TLS_C
 #define MBEDTLS_X509_CRT_PARSE_C
 #define MBEDTLS_X509_USE_C
+#define MBEDTLS_MD_C
```

Comment:
Not in config-suite-b.h but in crypto-config-suite-b.h, this is a crypto configuration option.